### ISSUE TITLE
PF6- still can not get Bad Conformer Id after the #510 issue fix 

### ISSUE THREAD

**Issue by wangyingxie** [rdkit/rdkit#5145]
**Configuration:**

- RDKit Version:'2021.09.2'
- Operating system: windows
- Python version (if relevant): 3.9.7
- Are you using conda? yes
- If you are using conda, which channel did you install the rdkit from? conda install rdkit
- If you are not using conda: how did you install the RDKit?

**Description: My rdkit version is latest one, so I assume the #510 is fixed. However, I still can not get the right conformer Id for PF6-, my code is shown as follow. 
The out put for list(Chem.GetPeriodicTable().GetValenceList(15)) is [3,5,7], I think it is permitted for PF6-, right?**

```
m = Chem.MolFromSmiles("F[P-](F)(F)(F)(F)F")
#Chem.AssignAtomChiralTagsFromStructure(m)
AllChem.EmbedMolecule(m, useRandomCoords=True,maxAttempts=5000)
AllChem.UFFOptimizeMolecule(m)
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_22084/332788942.py in <module>
      2 #Chem.AssignAtomChiralTagsFromStructure(m)
      3 AllChem.EmbedMolecule(m, useRandomCoords=True,maxAttempts=5000)
----> 4 AllChem.UFFOptimizeMolecule(m)

ValueError: Bad Conformer Id
```

**Comment by bp-kelley** [comment]
the full error from the above is:

```
>>> m = Chem.MolFromSmiles("F[P-](F)(F)(F)(F)F")
>>> AllChem.EmbedMolecule(m, useRandomCoords=True,maxAttempts=5000)
[13:27:40] UFFTYPER: Warning: hybridization set to SP3 for atom 1
[13:27:40] UFFTYPER: Unrecognized charge state for atom: 1
-1
```

So the forcefield can't properly type the molecule and the conformer returned is -1 which indicates a failure.
This looks to mean that the molecule currently can't be embedded.

**Comment by wangyingxie** [comment]
> UFFTYPER: Warning: hybridization set to SP3 for atom 1

Thanks for your notes. Then do you know how to deal with this issue?

**Comment by greglandrum** [comment]
The RDKit is currently not able to reliably generate conformer for octahedral species.
This will hopefully be fixed in a future release.

**Comment by wangyingxie** [comment]
> The RDKit is currently not able to reliably generate conformer for octahedral species.
> This will hopefully be fixed in a future release.

Thanks for letting me know.

**Comment by ytcs** [comment]
Any updates on this?

**Comment by ytcs** [comment]
Any updates on this?

**Comment by teltim** [comment]
What is the progress on this?

**Comment by wangyingxie** [comment]
Nope

**Comment by greglandrum** [comment]
It should be fixed in the 2022.03.2 patch release, which is due at the end of the month
